core/unit_test/TestTeamPolicyConstructors.hpp

```diff
@@ -0,0 +1,145 @@
+//@HEADER
+// ************************************************************************
+//
+//                        Kokkos v. 4.0
+//       Copyright (2022) National Technology & Engineering
+//               Solutions of Sandia, LLC (NTESS).
+//
+// Under the terms of Contract DE-NA0003525 with NTESS,
+// the U.S. Government retains certain rights in this software.
+//
+// Part of Kokkos, under the Apache License v2.0 with LLVM Exceptions.
+// See https://kokkos.org/LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//@HEADER
+
+#include <cstdio>
+#include <sstream>
+#include <iostream>
+
+#include <gtest/gtest.h>
+
+#include <Kokkos_Core.hpp>
+
+namespace {
+
+template <typename Policy>
+void test_run_time_parameters() {
+  int league_size = 131;
+
+  using execution_space = typename Policy::execution_space;
```

Comment:
```suggestion
  using ExecutionSpace = typename Policy::execution_space;
```
because snake case is used for variables everywhere else in this test.
(You can chose to ignore this suggestion)